PyInstaller/hooks/hook-PyQt5.QtQuick.py

```diff
@@ -32,18 +27,25 @@
         'QtQuick.2',
         #'QtSensors',
         #'QtTest'
+]
+
+qmldir = pyqt5_library_info.location['Qml2ImportsPath']
+# Per https://github.com/pyinstaller/pyinstaller/pull/3229#issuecomment-359735031,
+# not all PyQt5 installs have QML files. In this case, ``qmldir`` is empty.
+if not qmldir:
+    logger.warning('Unable to find Qt5 QML files. QML files not packaged.')
```

Comment:
This is related to trying to support the large number of PyQt5 packages out there. A "good" PyQt5 package will include QML, but not all do. Many PyQt5 applications will not use QML, but PyInstaller will still package it, since following the Qt library dependencies require it.

In summary: most people won't care, but a few might be affected, so I wanted to at least warn them.